src/js/stores/AppFormStore.js

```diff
@@ -295,16 +295,15 @@ function processResponseErrors(responseErrors, response, statusCode) {
       AppFormErrorMessages.getGeneralMessage("unknownServerError");
 
   } else if (statusCode === 422 && response != null &&
-      Util.isArray(response.errors)) {
-
-    response.errors.forEach((error) => {
-      var attributePath = error.attribute.length
-        ? error.attribute
+      Util.isArray(response.details)) {
```

Comment:
Ok, found the answer myself in the code and comments (not in the description) in the linked PRs.
Please be more descriptive next time in your PR description. Thank you.